src/styles/main.scss

```
@@ -0,0 +1,64 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
+  box-sizing: border-box;
+  padding: 10px;
+  gap: 1px;
+  top: 50%;
+  transform: translate(0, -50%);
+  left: 0;
+  right: 0;
+  margin: 0 auto;
+  max-width: $calendar-width;
+
+  $start-days: (mon 0, tue 101, wed 202, thu 303, fri 404, sat 505, sun 606);
```

```suggestion
  $start-days: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
```
you need to calculate margin-left in the `each`